homeassistant/components/remote_calendar/quality_scale.yaml

```diff
@@ -0,0 +1,82 @@
+rules:
+  # Bronze
+  config-flow: done
+  test-before-configure: done
+  unique-config-entry:
+    status: exempt
+    comment: |
+      No unique identifier.
+  config-flow-test-coverage: done
+  runtime-data: done
+  test-before-setup: done
+  appropriate-polling: done
+  entity-unique-id: done
+  has-entity-name: done
+  entity-event-setup:
+    status: exempt
+    comment: |
+      Entities of this integration does not explicitly subscribe to events.
+  dependency-transparency: done
+  action-setup:
+    status: exempt
+    comment: |
+      There are no actions.
+  common-modules: done
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions: done
+  docs-actions:
+    status: exempt
+    comment: |
+      No actions available.
+  brands: done
+  # Silver
+  config-entry-unloading: done
+  log-when-unavailable: done
+  entity-unavailable: done
+  action-exceptions: done
+  reauthentication-flow:
+    status: exempt
+    comment: |
+      There is no authentication required.
+  parallel-updates: done
+  test-coverage: done
+  integration-owner: done
+  docs-installation-parameters: done
```

Comment:
I would consider the URL an installation parameter that is not documented.  Essentially the idea is to give the user some idea of the information they are going to need when setting this up.

Reply:
https://github.com/home-assistant/home-assistant.io/pull/37568/commits/d77e060711a9fb05f9f2f93cb31023e376383eb4